includes/definitions/radware.yaml

```diff
@@ -0,0 +1,13 @@
+os: radware
+text: RADWARE
+type: network
+icon: radware
```

Comment:
Sorry, one last thing I've noticed is that the icon radware doesn't exist. You will also need to add this in:

http://docs.librenms.org/Developing/os/Initial-Detection/#icon-and-logo